src/igvfd/schemas/construct_library.json

```diff
@@ -0,0 +1,368 @@
+{
+    "title": "Construct library",
+    "$id": "/profiles/construct_library.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "description": "Schema for submitting a construct library.",
+    "type": "object",
+    "required": [
+        "award",
+        "lab",
+        "scope"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases",
+        "accession",
+        "alternate_accessions"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
```

Comment:
are we not mixing in any FileSet properties?

Reply:
Currently FileSet only has unique properties (non-mixins) Donors and Samples- do we want either of these?